dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresVectorStoreRecordPropertyMapping.cs

```diff
@@ -214,9 +213,6 @@ public static NpgsqlParameter GetNpgsqlParameter(object? value)
                         vectorIndexesToCreate.Add((dataProperty.StorageName, "", "", isVector: false));
                     }
                     break;
-
-                default:
```

Comment:
Any specific reason this needs to go? If the idea is to skip key properties, it's better to add a case for that with an explicit `continue` IMHO.

It's good to keep these in case we add other property model types in the future (I already have at least one in mind, for a nested record (hierarchical data models)).